.tugboat/config.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  drb-front-end:
+    image: tugboatqa/node:16
+    default: true
+    expose: 3000
+    commands:
+      build:
+        - npm install
+        - npm run build
+      start:
+        - npm run start &
```

Comment:
`start &`  ?  (It looks wrong but what do I know)

Reply:
It _is_ weird but it's a hack that Tugboat uses to get a service running in the background. Without it Tugboat thinks the start command hangs and never gives you the preview URL.

I don't like it but it's in their documentation!